src/xnnpack/AlignedAllocator.h

```diff
@@ -77,6 +77,14 @@ class AlignedAllocator {
       throw std::bad_alloc();
 #endif
     }
+#elif defined(_WIN32)
+    void* memory = nullptr;
+    memory = _aligned_malloc(n * sizeof(T), Alignment);
+    if (memory == 0) {
+#if !defined(__GNUC__) || defined(__EXCEPTIONS)
```

Comment:
The meaning of this macro is to remove the `throw` statement when the source C++ file is built without exceptions support. For Windows, you need to also check for MSVC exception macro, i.e.
```
#if !defined(__GNUC__) && !defined(_MSC_VER) || defined(__EXCEPTIONS) || defined(_CPPUNWIND)
```